pyproject.toml

```diff
@@ -17,3 +17,6 @@ exclude = '''
     dist
 )/
 '''
+
+[tool.pytest.ini_options]
+xfail_strict = true
```

Comment:
This could also be defined in the main setup.cfg, or even on a per-project basis - chose this file as it seemed like the preferred method for pytest configuration (after using a global pytest.ini which we currently don't have), but not sure if there's a better option.

Reply:
I agree that this is the best place for it. For reference, our guidance on which files to use is now codified in [our published developer guide](https://docs.rapids.ai/api/cudf/nightly/developer_guide/contributing_guide.html#code-style) (only nightly for now, will be live on stable docs in 22.10).